### ISSUE TITLE
How connection configuration procedure could be triggered?

### ISSUE THREAD

**Issue by abrilevskiy** [Jerrkawz/HAGearS3#20]
Hello,

I had to move my Home Assistance setup to another device and therefore IP and token are changed: could you please clarify how connection configuration procedure (token copy-paste via online service) could be triggered?

Thanks!

**Comment by jerrod-lankford** [comment]
You likely need to reinstall the app on your watch to trigger it

**Comment by samhed** [comment]
Yeah, reinstalling the app works for showing the link to the hassgalaxy.app page.
